engine/src/main/java/org/terasology/rendering/gltf/deserializers/Vector3fDeserializer.java

```diff
@@ -21,7 +21,7 @@
 import com.google.gson.JsonParseException;
 import gnu.trove.list.TFloatList;
 import gnu.trove.list.array.TFloatArrayList;
-import org.terasology.math.geom.Vector3f;
+import org.joml.Vector3f;
```

Comment:
Ehm, what is this deserializer used for? 

I would assume we keep all de- and encoding logic for the TeraMath data types and add new ones for the JOML types (if needed). Are we sure this should be changed?